chia/rpc/full_node_rpc_api.py

```diff
@@ -835,6 +838,38 @@ async def get_mempool_items_by_coin_name(self, request: dict[str, Any]) -> Endpo
 
         return {"mempool_items": [item.to_json_dict() for item in items]}
 
+    async def create_block_bundle_from_mempool(self, _: dict[str, Any]) -> EndpointResult:
```

Comment:
I think the "from mempool" part of this name is implied

Reply:
Fixed